#External Prior/Transform Input
extrinsics:
  baseLinkFrame: BASE
  lidarFrame: os_sensor        #LiDAR frame name - used to lookup LiDAR-to-ExternalSensor Frame
  imuBaseFrame: IMU_link
  cabinFrame: CABIN
  gnssFrame1: GNSS_L
  gnssFrame2: GNSS_R
